.github/workflows/test.yml

```diff
@@ -10,6 +10,8 @@ jobs:
   linux-unit-tests:
     needs: yarn-lockfile-check
     uses: salesforcecli/github-workflows/.github/workflows/unitTestsLinux.yml@main
+    with:
+      skipTsDepCheck: true
```

Comment:
opt-in way to disable the ts-dep-check, see: https://github.com/salesforcecli/github-workflows/pull/120

this is a devDep so it's ok to ship TS